x-pack/plugins/cloud_security_posture/public/common/hooks/use_navigate_findings.ts

```diff
@@ -57,7 +57,7 @@ const useNavigate = (pathname: string, dataViewId = SECURITY_DEFAULT_DATA_VIEW_I
   const { services } = useKibana();
 
   return useCallback(
-    (filterParams: NavFilter = {}) => {
+    (filterParams: NavFilter = {}, groupBy?: string[]) => {
```

Comment:
might be to read and use if this function will receive an object, not urgent since it will require some refactoring